client/starwhale/core/dataset/model.py

```diff
@@ -121,6 +121,7 @@ def __init__(self, uri: URI) -> None:
             str, t.Any
         ] = {}  # TODO: use manifest class get_conda_env
         self.yaml_name = DefaultYAMLName.DATASET
+        self._version = uri.object.version if uri.object and uri.object.version else ""
```

Comment:
https://github.com/star-whale/starwhale/blob/main/client/starwhale/base/uri.py#L16 
If the uri is URI type, the uri.object is not None, and the uri.object.version default value is ""

```
self._version = uri.object.version if uri.object and uri.object.version else ""
```
-->
```
self._version = uri.object.version
```

Reply:
done